src/components/common/table.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+interface TableProps {
+	headers: string[];
+	data: any[];
+}
+
+const Table: React.FC<TableProps> = ({ headers, data }) => {
```

Comment:
Please don't make your own table, table components are complicated and we already have a table component